packager/certcache/certcache.go

```diff
@@ -396,6 +411,10 @@ func waitForSpecifiedTime(waitTimeInMinutes int, numRetries int) int {
 	waitTimeDuration := time.Duration(waitTimeInMinutes) * time.Minute
 	// For exponential backoff.
 	newWaitTimeInMinutes := 2 * waitTimeInMinutes
+	if newWaitTimeInMinutes > 10 {
+		// Cap the wait time at 10 minutes.
+		newWaitTimeInMinutes = 10
```

Comment:
Per this change, the comment on maxOCSPTries needs to be updated.

Reply:
Will address in follow-up PR #369